source/systems/gaea_user_guide.rst

```diff
@@ -1793,7 +1793,7 @@ number identified in Step 1 or as needed):
 .. code-block:: shell
 
   ssh-LXXXXX:localhost:XXXXX
-  First.Last@gaea-rsa.princeton.rdhpcs.noaa.gov
+  First.Last@gaea-rsa.rdhpcs.rdhpcs.noaa.gov
```

Comment:
Remove the duplicate "rdhpcs" -- should be `First.Last@gaea-rsa.rdhpcs.noaa.gov"